User Guide 3.2/Items/math-expressions.md

```diff
@@ -0,0 +1,24 @@
+<!--
+created_at: 2017-08-30
+authors:         
+    - "Catherine Pease"
+--> 
+
+# Math Expressions
+
+>Math Expressions - ie. mathematical operators - can be employed in interactions by using the Math Editor. The *Math Editor* is a WYSIWYG editor based on MathQuill (http://mathquill.com/), which allows you to use mathematical symbols to create LaTeX expressions containing mathematical operators. It is found in the Custom Interactions section.
+
+To access the Math Editor follow the steps below.
+
+**1.**  Once you have created a new Item, click on the [Custom Interactions library](../appendix/glossary.md#custom-interactions-library) below *Graphic Interactions* on the left, and drag the *Math Entry* icon onto the blank Item and drop it onto the [canvas](../appendix/glossary.md#canvas).
+
+A list of mathematical symbols will appear, with an empty text field below.
+
+![Math Editor](../resources/backend/items/authoring/features/math-editor.png)
+
+**2.** Click on the mathematical symbols you wish to use.
+
+These will appear in the text field. They can then be used in the writing of mathematical formulae, such as questions on geometry.
```

Comment:
Here is just a small Typo, 'formulae' instead of 'formula'

Reply:
formulae is the plural form of formula

Reply:
Oh.. sorry :+1:  learning

Reply:
that's ok :)